packages/story-editor/src/components/colorPicker/karma/colorPicker.karma.js

```diff
@@ -52,9 +52,12 @@ describe('ColorPicker', () => {
         // Click the background page panel color preview
         await fixture.events.click(bgPanel.backgroundColor.button);
 
-        await waitFor(() =>
-          expect(bgPanel.backgroundColor.picker).toBeDefined()
-        );
+        await waitFor(() => {
+          if (!bgPanel.backgroundColor.picker) {
+            throw new Error('picker not ready');
+          }
+          expect(bgPanel.backgroundColor.picker).toBeDefined();
+        });
```

Comment:
💭  What are your thoughts on changing the queries in the containers themselves to be `findBy` but use a max timeout of `5000ms`?

That way when people use these, they won't have to worry about making sure they are using `waitFor` correctly?

![image](https://user-images.githubusercontent.com/22185279/152193372-ce1a0f40-81f6-4b55-8641-3382bed5b5cb.png)

Reply:
I like the idea.